Apps/W1/EDocumentConnectors/FORNAV/app/src/Permissions EDoc/ForNAVEDocSetup.PermissionSet.al

```diff
@@ -0,0 +1,15 @@
+namespace Microsoft.EServices.EDocumentConnector.ForNAV;
+using System.Security.AccessControl;
+
+permissionset 6414 "ForNAV EDoc Setup"
+{
+    Access = Internal;
+    Assignable = true;
+    IncludedPermissionSets = LOGIN, "D365 BASIC", Microsoft.eServices.EDocument."E-Doc. Core - Basic", SUPER;
+    Permissions =
+        table * = X,
```

Comment:
Are these all tables, or just tables in the app?
Regardless its not a pattern we use.